ui/src/containers/VolumeInformation.js

```diff
@@ -82,6 +92,23 @@ const VolumeInformation = props => {
           ]}
         />
       </BreadcrumbContainer>
+      <VolumeInformationTitle>
+        {intl.messages.detailed_information}
+      </VolumeInformationTitle>
+
+      {volume?.status?.phase === STATUS_FAILED ? (
+        <Banner
+          type={STATUS_BANNER_ERROR}
+          icon={<i className="fas fa-exclamation-triangle" />}
+          title={
+            volume?.status?.errorCode === 'CreationError'
```

Comment:
I could give you an exhaustive list of error codes so that you can map each to a string (instead of only `CreationError`) but let's keep it like that => this will be reworked when we move to `conditions` (and I plan to work on it next week).